src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(method, number) {
```

Comment:
The `operate` method expects a function as the `method` argument. It would be helpful to document this requirement or add validation to ensure that the `method` is a function.